pyfr/plugins/fluidforce.py

```diff
@@ -8,9 +8,7 @@
 from pyfr.plugins.base import BasePlugin, init_csv
 
 
-class FluidForcePlugin(BasePlugin):
-    name = 'fluidforce'
-    systems = ['euler', 'navier-stokes']
+class BaseFluidForcePlugin(BasePlugin):
```

Comment:
Lets leave this plugin out of the first version of this pull request.  We can sort it out later.